package.json

```diff
@@ -21,24 +21,25 @@
   },
   "devDependencies": {
     "chai": "~3.5.0",
-    "coffee-coverage": "~1.0.1",
-    "coffee-script": "^1.12.6",
-    "coffeelint": "^1.16.0",
     "grunt": "~1.0.1",
     "grunt-contrib-watch": "~1.0.0",
     "grunt-mocha-test": "~0.7.0",
     "grunt-release": "~0.14.0",
-    "hubot": "2.x",
-    "hubot-mock-adapter": "~1.0.0",
-    "istanbul": "~0.4.5",
+    "hubot": "hubotio/hubot#fix-parse-help",
```

Comment:
what we should do is release `hubot@3.0.0` with dist-tag `@next`. That way others will not get it installed by default, but we can set `^3.0.0` here so it will already work.